### ISSUE TITLE
Activation of remote harvesting possible if balance 0 and pending incoming TX

### ISSUE THREAD

**Issue by fyahfox** [NewEconomyMovement/NemCommunityClient#319]
0.5.18-BETA

If you have 0 balance in an account you can't activate remote harvesting (which is fine of course, because the activation is a transaction and therefore costs a fee).

If you now send some XEM to the account, you can activate remote harvesting for that account even if the incoming transaction is still pending! I think that should not work.

**Comment by BloodyRookie** [comment]
Confirmed.

**Comment by fyahfox** [comment]
RIght, it is no threat to the system, but I think a pessimistic approach would be best:
IF 'balance with pending tx included" OR 'balance without pending tx included' < 'tx+fees' (for a current transaction)
THEN warning is displayed: not enough balance (or whatever that warning is right now)

**Comment by BloodyRookie** [comment]
Maybe we change it at some time after launch but not before launch because that would break our design.

**Comment by fyahfox** [comment]
sure, this is no high priority problem :)
